CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Unity Converters for Newtonsoft.Json changelog
+
+## x.0.1
+
+- ✨ Initial release
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5097765)